frontend/components/transaction/BidTransactionTable.vue

```diff
@@ -0,0 +1,182 @@
+<template>
+    <div class="flex flex-col space-y-1">
+        <div class="flex justify-between">
+            <div>Auction Ends</div>
+            <div>
+                <time-till :date="auctionTransaction.endDate" />
+            </div>
+        </div>
+        <div class="flex justify-between">
+            <div>Auction Amount</div>
+            <div>
+                <FormatCurrency
+                    :value="auctionTransaction.collateralAmount"
+                    :currency="auctionTransaction.collateralSymbol"
+                />
+            </div>
+        </div>
+        <div class="flex justify-between">
+            <div>Auction Price</div>
+            <div>
+                <template v-if="auctionTransaction.isActive">
+                    <PriceDropAnimation :auction="auctionTransaction" class="mr-1" />
+                    <FormatCurrency :value="auctionTransaction.approximateUnitPrice" currency="DAI" /> per
+                    <span class="uppercase">{{ auctionTransaction.collateralSymbol }}</span>
+                </template>
+                <span v-else class="opacity-50">Unknown</span>
+            </div>
+        </div>
+        <div class="flex justify-between">
+            <div>
+                Transaction Fee
+                <span class="text-gray-300">
+                    (~
+                    <FormatCurrency
+                        v-if="auctionTransaction.biddingTransactionFeeETH"
+                        :value="auctionTransaction.biddingTransactionFeeETH"
+                        :decimals="5"
+                    />
+                    <span v-else>Unknown</span>
+                    ETH)
+                </span>
+            </div>
+            <div>
+                <FormatCurrency
+                    v-if="auctionTransaction.biddingTransactionFeeDAI"
+                    :value="auctionTransaction.biddingTransactionFeeDAI * -1"
+                    currency="DAI"
+                />
+                <span v-else class="opacity-50">Unknown</span>
+            </div>
+        </div>
+        <div class="flex justify-between">
+            <button
+                :class="{ 'ClickableText underline': isActive && auctionTransaction.totalPrice }"
+                @click="setAmountToBid(undefined)"
+            >
+                Buy all collateral
+            </button>
+            <button
+                :class="{ ClickableText: isActive && auctionTransaction.totalPrice }"
+                @click="setAmountToBid(undefined)"
```

Comment:
It's better to use html-native `disabled` attribute, which will disable the click event for you (so you don't have to check it inside the method)
```suggestion
                class="Clickable"
                :disabled="!isActive || !auctionTransaction.totalPrice"
                @click="setAmountToBid(undefined)"
```
In css you can always use:
```css
.ClickableText:enabled {

}
```
or `.ClickableText:not([disabled])` to apply style only when the button is not disabled

Reply:
(same for 3 other places in this component)